src/dry-runner.js

```diff
@@ -0,0 +1,136 @@
+import childProcess from "child_process";
+import {Promise} from "es6-promise";
+import fileUtils from "./file-utils";
+import fs from "fs";
+import logger from "./logger";
+import mockfs from "mock-fs";
```

Comment:
Doesn't merely importing this start the monkey-patching of node internals? I think we want a lazy `require` in here right before we actually need to start doing stuff...